### DISCUSSION TITLE
TeFlow Public Weights - Local Evaluation Screenshot

### DISCUSSION THREAD

**Discussion by Kin-Zhang** [Kin-Zhang/TeFlow#3] [Show and tell]
Agoverse 2:
<img width="1041" height="857" alt="image" src="https://github.com/user-attachments/assets/4e583239-45f2-4f06-a455-f799fb078644" />

nuScenes:
<img width="1054" height="832" alt="image" src="https://github.com/user-attachments/assets/3210cb53-331a-4592-b3fb-cd49944903a3" />

Waymo:
<img width="1045" height="857" alt="image" src="https://github.com/user-attachments/assets/da1b9690-6913-460d-ad6a-1d56d10f3bf6" />

**Comment by yanconglin** [comment]
Hey, thank you for sharing this data.  I would appreciate a clarification on the results:  the numerical values posted there differ from the ones in the TeFlow paper (nuScenes and Waymo). I was wondering what leads to this disparity and which result to use?

**Reply by Kin-Zhang** [reply]
The difference likely stems from two factors:

1. **Machine/environment differences**: Numerical results can vary slightly across different hardware and software environments. For the most reliable and reproducible numbers, we recommend downloading the checkpoint and evaluating it on your own machine.
2. **Different evaluated frame.** (maybe): The evaluated frames may be updated, but I cannot validate since the old ones were cleaned by me at some point due to a full disk. :(
3. **Ground segmentation updates**: The ground mask used in our evaluation pipeline has been updated since the paper was submitted. When setting up the new leaderboard, we overwrote the previous ground mask, which can affect the reported metrics. For reference, all baseline results for the nuScenes and Waymo experiments in the TeFlow paper were re-run at the same time, using the same ground mask, on the same machine — so those results are internally consistent.



In short, if you are looking to make a fair comparison against the TeFlow, we suggest running the evaluation in the same setups with your methods (data, machine).

btw, this time you can also upload your result file to the public online leaderboard, where we will report five datasets' results together. (and teflow results on 0-35m also reported there: https://sceneflow.argoverse.org/leaderboard)

**Reply by yanconglin** [reply]
Currently, most leaderboard models are trained on AV2, making it a convenient benchmark for evaluating models and loss functions trained on the same dataset. I am interested in knowing the performance gap when training on alternative datasets such as Waymo and nuScenes.

**Comment by Kin-Zhang** [comment]
Do you mean domain-gap? or in-domain results difference?

- For domain gap, there are some results discussion in deltaflow, uniflwo,
synflow. and teflow-av2 in leaderboard looks good score also in the other
two datasets. (so it might already present something to you there)
- For in-domain results difference, I think you can upload teflow-waymo
ckpt results into new leaderboard to have a look if you are interested.

**Reply by yanconglin** [reply]
Something else: would it be possible to access the SyncFlow pretrained weights at this moment? (which has showcased the value of synthetic data).

**Comment by Kin-Zhang** [comment]
Sure, I can try to upload datasets and ckpt recently. Which ckpt do you
need for the moment? I can share a quick Google Drive link for you today.

**Reply by yanconglin** [reply]
great! Would you mind sharing the model trained on synthetic data only (SynFlow-4k, DeltaFlow backbone)? many thanks

**Comment by Kin-Zhang** [comment]
https://drive.google.com/file/d/1ZqPTFdZUVVS152hnbey-rYc9vZvpCpt6/view?usp=drive_link.
Please send a request through this list. I uploaded the ckpt for this
item's results (SynFlow-4k only in DeltaFlow backbone):
3 SynFlow-4k unsupervised [link] <https://kin-zhang.github.io/SynFlow/>
KTH-RPL 0.2566 0.1279 0.1190 0.2523 0.1784 0.2720 2026-04-10
